packages/workbox-webpack-plugin/test/node/e2etest.js

```diff
@@ -0,0 +1,84 @@
+/* global workbox, expect, describe */
+const assert = require('chai').assert;
+const WorkboxBuild = require('../../../workbox-build/src');
+const proxyquire = require('proxyquire');
+const webpack = require('webpack');
+const fs = require('fs');
+const path = require('path');
+
+let WorkboxWebpackPlugin;
+
+function runWebpack(webpackConfig) {
+  return new Promise((resolve, reject) => {
+    let compiler = webpack(webpackConfig);
+    compiler.run((err, stats)=>{
+      if (err) reject(err);
+      else {
+        // Timeout for plugins that work on `after-emit` event of webpack
+        setTimeout(()=>{
+          resolve(stats);
+        }, 20);
+      }
+    });
+  });
+}
+
+describe('Tests for webpack plugin', function() {
+  this.timeout(55000);
+  beforeEach(()=>{
+    // do a proxy require
+    WorkboxWebpackPlugin = proxyquire('../../', {
+      'workbox-build': WorkboxBuild,
```

Comment:
workbox-build will be "npm linked" with the local workbox-build through lerna. So you only need to have workbox-box as a dependency in your package.json and then require as normal.